app/services/processors/pull_request_size_updater.rb

```diff
@@ -0,0 +1,9 @@
+module Processors
+  class PullRequestSizeUpdater < BaseService
+    def call
+      Events::PullRequest.find_each(batch_size: 100).lazy.each do |pull_request|
```

Comment:
Interesting! Just out of curiosity: Why have we chosen such a small batch size? What leverage does `lazy` provides here?

Reply:
Good point, that batch size is indeed too low. I'd better use the default size of 1000 instead.
As for `lazy`... it's a good question. I remember @hdamico suggesting me to add it in another query some time ago, so I thought it would be suitable here too. But yeah, I'm not entirely sure what difference it makes.